src/prefect/cli/deployment.py

```diff
@@ -411,31 +412,44 @@ async def run(
         )
     parameters = {**multi_params, **cli_params}
 
-    # Parse the start time if provided
-    if start_in:
-        start_time_raw = "in " + start_in
-    elif start_at:
-        start_time_raw = "at " + start_at
-    else:
-        start_time_raw = None
+    if start_in and start_at:
+        exit_with_error(
+            "Expected optional start_in field or start_at field but not both."
+        )
 
-    if start_time_raw:
+    elif start_in is None and start_at is None:
+        scheduled_start_time = now
+        human_dt_diff = " (now)"
+    else:
+        if start_in:
+            start_time_raw = "in " + start_in
+        else:
+            start_time_raw = "at " + start_at
```

Comment:
Since the above exits if both are set you can just do
```python
if start_in: ... 
elif start_at: ...
else: ...
```

Reply:
Do you mean `else:` being where `start_in` is None and `start_as` in None? Because there is some logic beneath that specific to `start_in`/`start_at` for assigning `start_time_parsed`, `scheduled_start_time`, and `human_dt_diff`.

Reply:
Yeah I think you can do `else` instead of `elif start_in is None and start_at is None:`

Reply:
I'm not sure what you mean, since the `else` on ln 423 handles cases where `start_at` or `start_in` is set. If neither are set, we wouldn't want to do anything from ln 423-452.

Reply:
I'm kinda of curious what this looks like to without having a helper function for the raw parsing logic.

Reply:
I think the helper function is a bit harder to read than the initial implementation, especially since it does more than one thing (parses the date time _and_ generates a formatted diff). I don't have strong feelings here though.

Reply:
(unresolving for visibility; does not mean it needs to be addressed)